model/src/gateway/payload/member_update.rs

```diff
@@ -9,6 +9,8 @@ pub struct MemberUpdate {
     pub guild_id: GuildId,
     pub joined_at: String,
     pub nick: Option<String>,
+    #[serde(default)]
+    pub pending: bool,
```

Comment:
Also, should this be an `Option<bool>` to sort of denote a state change, to support use-cases such as adding roles after passing membership screening?